packages/react-link/src/components/Link/Link.test.tsx

```diff
@@ -19,12 +15,68 @@ describe('Link', () => {
     }
   });
 
-  /**
-   * Note: see more visual regression tests for Link in /apps/vr-tests.
-   */
-  it('renders a default state', () => {
-    const component = renderer.create(<Link>Default Link</Link>);
+  isConformant({
+    Component: Link,
+    displayName: 'Link',
+  });
+
+  describe('AccessibilityLinkBehavior', () => {
+    const testFacade = new ComponentTestFacade(Link, {});
+    const errors = validateBehavior(linkBehaviorDefinition, testFacade);
+    expect(errors).toEqual([]);
+  });
+
+  it('renders as a button if no href is provided', () => {
+    wrapper = mount(<Link>This is a link</Link>);
```

Comment:
this is not a must for me, but using @testing-library/react would be nice :)

Reply:
I'd like to make the move to `@testing-library/react` if we decide on it in a follow up PR if that's ok